npm/test-lint.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+// ---------------------------------------------------------------------------------------------------------------------
+// This script is intended to contain all actions pertaining to code style checking, linting and normalisation.
+//
+// 1. The script executes linting routines on specific folders.
+// ---------------------------------------------------------------------------------------------------------------------
+
+require('shelljs/global');
+require('colors');
+
+var async = require('async'),
+    ESLintCLIEngine = require('eslint').CLIEngine,
+
+    LINT_SOURCE_DIRS = [
+        './test',
+        './index.js',
+        './lib',
```

Comment:
I figured once that eslint does not go recursive unless one does `/lib/**/*.js` can u confirm @kunagpal ?